components/mdx/mdx-client.tsx

```
@@ -0,0 +1,93 @@
+export const Callout = (props: BlockquoteProps) => {
+  return (
+    <blockquote
+      className="my-4 border-l-4 border-blue-500 bg-blue-50 p-4 dark:bg-gray-800 dark:border-blue-400"
+      {...props}
+    />
+  );
+};
+
+export const Code = ({ children, ...props }: ComponentPropsWithoutRef<"code">) => {
+  const codeHTML = highlight(children as string);
+  return <code dangerouslySetInnerHTML={{ __html: codeHTML }} {...props} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)